content/blog/2023/05/10/2023-05-10-jenkins-april-newsletter.adoc

```diff
@@ -0,0 +1,144 @@
+*** Adding `ppc64le` support back into the Jenkins CI agent Docker build. (link:https://github.com/jenkinsci/docker-agent/pull/391[#391]) link:https://github.com/ksalerno99[@ksalerno99]
+*** Exposing the default image user to the environment variable user. (link:https://github.com/jenkinsci/docker-agent/pull/400[#400]) link:https://github.com/dduportal[@dduportal]
+* author:markewaite[Mark Waite] is working on a link:https://github.com/jenkinsci/jenkins/pull/7913[system] that would warn when operating system end-of-life is approaching.
+
+[[documentation]]
+image:/images/post-images/2023/02/07/2023-02-07-jenkins-newsletter/documentation.png[Documentation Update]
+Contributed by: author:kmartens27[Kevin Martens]
+
+Over the course of April, there were 4 blog posts published, featuring seven different authors.
+author:gounthar[Bruno Verachten] continues his series on building android apps in Jenkins.
+As Google Summer of Code begins, we want to acknowledge and thank all of the applicants for their efforts.
+Thanks to all of the continuing and new contributors, all of your work helps support both the Jenkins project and the Open-Source community.
+
+We also want to thank DigitalOcean for their continued support and sponsorship of Jenkins.
+They have provided us with an additional $8,400 credit as the Infrastructure team works on reducing bandwidth usage further.
+
+[[outreach]]
+image:/images/post-images/2023/01/12/jenkins-newsletter/outreach-and-advocacy.png[Outreach and advocacy Update]
+
+Contributed by: author:alyssat[Alyssa Tong]
+
+*Jenkins in Google Summer of Code (GSoC)*
+
+We had an unprecedented number of GSoC applicants interested in Jenkins this year.
+The Jenkins project received over 60 proposals by the close of the application period.
+Dedicated Jenkins mentors worked overtime and weekends to review and grade proposals within a two weeks period.
+Many THANKS to the wonderful Jenkins mentors, this program isn’t possible without them.
+
+Best of luck to all GSoC participants!
+
+*Jenkins at cdCon*
+
+image:/images/post-images/2023/05/10/2023-05-10-jenkins-april-newsletter/image1.png[Jenkins talks at cdCon,width=269,height=140]
+
+cdCon took place on May 8 - 9, 2023 in Vancouver, Canada as link:https://events.linuxfoundation.org/cdcon-gitopscon/[cdCon + GitOpsCon], co-organized with the link:https://cncf.io/[Cloud Native Computing Foundation (CNCF)].
+Members of the Jenkins Governance Board, long-time Jenkins users, and contributors author:markewaite[Mark Waite] and author:notmyfault[Alex Brandes] were in attendance.
+Mark partcipiated in link:https://sched.co/1Js9F[the Graduated Projects Keynote Panel], discussing Jenkins Community’s experiences with graduation and sharing his thoughts on why graduation matters for the community and users of Jenkins.
+
+If you attended cdCon, we hope you had a fantastic time!
```

Comment:
Per @kmartens27 feedback. I think we can just remove this section from the newsletter.